COLLABORATOR_GUIDE.md

```diff
@@ -403,10 +403,11 @@ recommended but not required.
 
 ### Deprecations
 
-_Deprecation_ refers to the identification of Public APIs that should no longer
-be used and that may be removed or modified in backward-incompatible ways in
-a future major release of Node.js. Deprecation may be used with internal APIs if
-there is expected impact on the user community.
+__Deprecation__ refers to the identification of Public APIs that should no
+longer be used.
+
+Once specific bits of code are marked as being Deprecated, they are no longer
```

Comment:
`specific bits of code are marked as being Deprecated` -> `an API is deprecated`